src/state/shellscripts/bwrap.sh

```diff
@@ -1,9 +1,4 @@
-#!/usr/bin/env bash
-
-set -ue
```

Comment:
Could you leave `set -ue` as is? Moving it to the shebang line does not set them if the script is called using `sh` directly (e.g. `sh ./bwrap.sh`) and thus has a different behaviour.